apps/store/extensions/app/social-reviews/modules/reviews-utils.js

```diff
@@ -43,37 +46,71 @@ var ReviewUtils = {};
         paging.limit = paging.limit ? paging.limit : REVIEW_PAGE_LIMIT;
         return paging;
     };
-    var processUserReviews = function(socialSvc, user, reviews) {
+    var processUserReviews = function (socialSvc, user, reviews, target) {
         var review;
-        var formattedUsername = formatUsername(user);
+        var myReview;
+        var socialSvc = getSocialSvc();
+        var username = formatUsername(user);
+        myReview = JSON.parse(String(socialSvc.getUserComment(username, target)));
+        if (!isEmpty(myReview)) {
```

Comment:
AFAIU there is a possibility to be myReview=undefined. If so what will happened to !isEmpty(myReview). Please check getUserComment.